app/js/lib/foxbox/services/base.js

```diff
@@ -232,7 +237,7 @@ export default class BaseService extends EventDispatcher {
    * super.teardown() method as well.
    */
   teardown() {
-    for (let watcher of this[p.watchers].values()) {
+    for (const watcher of this[p.watchers].values()) {
```

Comment:
Does it work natively on Firefox? I think we're now transpiling `const` and `let` so that's not really an issue.